streamer/src/nonblocking/quic.rs

```diff
@@ -205,20 +237,25 @@ async fn setup_connection(
         if let Ok(new_connection) = connecting_result {
             stats.total_connections.fetch_add(1, Ordering::Relaxed);
             stats.total_new_connections.fetch_add(1, Ordering::Relaxed);
-            let NewConnection {
-                connection,
-                uni_streams,
-                ..
-            } = new_connection;
 
-            let remote_addr = connection.remote_address();
+            let remote_addr = new_connection.connection.remote_address();
             let mut remote_pubkey = None;
 
             let table_and_stake = {
```

Comment:
Looks like `table_and_stake` is not used and it will be removed in follow up commits